src/utils/npm.js

```diff
@@ -39,10 +39,24 @@ export function publish(
 ) {
   return npmRequestLimit(async () => {
     logger.info(messages.npmPublish(pkgName));
-    let publishFlags = opts.access ? ['--access', opts.access] : [];
-    return await processes.spawn('npm', ['publish', ...publishFlags], {
-      cwd: opts.cwd
-    });
+    const publishFlags = opts.access ? ['--access', opts.access] : [];
+
+    try {
+      const result = await processes.spawn(
+        'npm',
+        ['publish', ...publishFlags],
+        {
+          cwd: opts.cwd
+        }
+      );
+      return { published: true, publishedPkgInfo: JSON.parse(result.stdout) };
+    } catch (error) {
+      //Publish failed
+      if (error.code == 0) {
```

Comment:
I would remove this check too, and just return false (Not sure if you can return false directly, might need to wrap it in a promise 

```
return Promise.resolve(true);
```

We can check tomorrow